frontend/src/languages/en.json

```diff
@@ -34,7 +34,8 @@
     "login.msg.password.missing": "A password is required",
     "login.notice.message": "Notice: Access to this service is for authorized personnel only. If you do not have the expressed authorization of the administrator, you must exit now. This organization prohibits unauthorized access, disclosure, duplication, modification, diversion, destruction, loss, misuse, or theft of its information.",
     "error.invalidcredentials": "Username or Password are incorrect",
-    "patient.label.modify" : "Add/Modify Patient" ,
+    "patient.label.modify" :"Add Or Modify Patient" ,
+    "patient.label.PatientHistory" :"PatientHistory" ,
```

Comment:
Patient History are two words . 
but the property already exists here https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop_3x/frontend/src/languages/en.json#L42

Reply:
It was an Oversight. Resolved.